### PR TITLE
drop django 1.9, add django 1.11 and django dev

### PR DESCRIPTION
**Pull request by lynnco**

### CODE REVIEW

**Comment by lynnco** [comment]
includes some of the django 2.0 changes that won't break 1.8

File: .travis.yml [hunk 1]
```diff
@@ -4,20 +4,33 @@ python:
...
+          env: TOXENV=django111
+        - python: "2.7"
+          env: TOXENV=djangodev
+        - python: "3.3"
+          env: TOXENV=djangodev
+        - python: "3.4"
+          env: TOXENV=djangodev
+        - python: "3.5"
+          env: TOXENV=djangodev
+    allow_failures:
```
Comment (kelseyq) [inline .travis.yml]: cool!

File: callisto/delivery/models.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ def _unpepper(peppered_report):
 class Report(models.Model):
     """The full text of a reported incident."""
     encrypted = models.BinaryField(blank=False)
-    owner = models.ForeignKey(settings.AUTH_USER_MODEL)
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```
Comment (kelseyq) [inline callisto/delivery/models.py]: do we have tests of the delete behavior?

Comment (lynnco) [inline callisto/delivery/models.py]: we don't, `on_delete=models.CASCADE` is the default behavior, this line just makes it explicit

Comment (lynnco) [inline callisto/delivery/models.py]: cite: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ForeignKey.on_delete

Comment (kelseyq) [inline callisto/delivery/models.py]: fair enough, but i think adding tests to demonstrate delete behavior would be a good idea given how sensitive the data is. don't think it needs to be in this PR though-- added #178 to track it

**Pull request merged**